Makefile

```diff
@@ -3,21 +3,23 @@
 THIS_FILE := $(lastword $(MAKEFILE_LIST))
 CURRENT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
-#################################
+############################
 # Make targets for dsp-tools
-#################################
-
-.PHONY: clone-dsp-repo
```

Comment:
We don't need an extra command to clone the DSP-API. I included this in the `dsp-stack` command.